src/helpers/fallbackauthdialog.ts

```diff
@@ -8,6 +8,25 @@
 
 import { Configuration, PublicClientApplication, RedirectRequest } from "@azure/msal-browser";
 
+const msalConfig: Configuration = {
+  auth: {
+    clientId: "{application GUID here}", //This is your client ID
+    authority: "https://login.microsoftonline.com/common",
+    redirectUri: "https://localhost:{PORT}/fallbackauthdialog.html",
```

Comment:
Is this `PORT` a variable? Or just some value for the client to change?

Reply:
This was just moved from another place in the file with these already that way.  You're forgetting this is the sso template the requires the extra stop of 'npm run configure-sso' which updates these values with unique and good ones.